raiden/network/matrixtransport.py

```diff
@@ -364,11 +363,11 @@ def _handle_message(self, room, event):
             )
         elif isinstance(message, SignedMessage):
             self._receive_message(message)
-        elif log.isEnabledFor(logging.ERROR):
-            log.error(
-                'Invalid message',
-                message=data,
-            )
+
+        log.error(
+            'Invalid message',
+            message=data,
+        )
```

Comment:
This needs to be in an `else` block, instead of after `if/else` block, or it'll log `Invalid message` even for messages handled by the methods above.

Reply:
Argh yes ... thanks for spotting it.